docs/user_guide.rst

```diff
@@ -277,22 +292,42 @@ The below table summarises the currently available options:
      - Type
      - Example value
      - Description
-   * - `calibration_set_id`
-     - str
+   * - :attr:`calibration_set_id`
+     - :class:`uuid.UUID`
      - "f7d9642e-b0ca-4f2d-af2a-30195bd7a76d"
-     - Indicates the calibration set to use. Defaults to `None`, which means the IQM server will use the best
+     - Indicates the calibration set to use. Defaults to ``None``, which means the IQM server will use the best
        available calibration set automatically.
-   * - `max_circuit_duration_over_t2`
-     - float
+   * - :attr:`compiler_options`
+     - :class:`~iqm.iqm_client.models.CircuitCompilationOptions`
+     - see below
+     - Contains various options that affect the compilation of the quantum circuit into an
+       instruction schedule.
+
+The :class:`~iqm.iqm_client.models.CircuitCompilationOptions` class contains the following attributes (in addition to some
+advanced options described in the API documentation):
+
+.. list-table::
+   :widths: 25 20 25 100
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Example value
+     - Description
+   * - :attr:`max_circuit_duration_over_t2`
+     - :class:`float` | :class:`None`
      - 1.0
      - Set server-side circuit disqualification threshold. If any circuit in a job is estimated to take longer than the
        shortest T2 time of any qubit used in the circuit multiplied by this value, the server will reject the job.
        Setting this value to ``0.0`` will disable circuit duration check.
        The default value ``None`` means the server default value will be used.
-   * - `heralding_mode`
-     - :py:class:`~iqm_client.models.HeraldingMode`
+   * - :attr:`heralding_mode`
+     - :class:`~iqm.iqm_client.models.HeraldingMode`
```

Comment:
I was wondering why we used `ZEROS` and not `ZEROES` since most of our docs tend to use British English :)

Reply:
We haven't standardized the spelling everywhere I guess.